src/sentry/incidents/apps.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import
+
+from django.apps import AppConfig
+
+
+class IncidentsAppConfig(AppConfig):
+    name = "sentry.incidents"
+
+    def ready(self):
+        from . import receivers  # noqa
```

Comment:
@wedamija Ah dang, I just realized that this is going to be broken since something (not covered by tests) will expect sentry.incidents.receivers.

Reply:
Not sure what you mean? I don't think anything expects that to be imported in `__init__` (I wrote this code recently and would have explicitly avoided that)

Reply:
Sorry, `s/this is going to/this might be`. I just looked, don't see any references. Should I `pass` then?

Reply:
Oh, it needs to be imported in the `AppConfig` because it registers signal handlers. But it doesn't need to be imported into the `__init__` space, nothing is relying on it happening there.